instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/cache/concurrentlinkedhashmap/NOTICE

```diff
@@ -0,0 +1,7 @@
+ConcurrentLinkedHashMap
+Copyright 2008, Ben Manes
```

Comment:
I think we should add this line above the Copyright 2010 Google Inc. into the file license headers. We could probably remove the NOTICE then (otherwise I would put the NOTICE in resources, not java)

Reply:
By the way I noticed micronaut doesn't reference the NOTICE, nor update license headers

https://github.com/micronaut-projects/micronaut-core/tree/813923882bcfed158d136a27bf8010edf70bb613/core/src/main/java/io/micronaut/core/util/clhm

IANAL, but I think the `author` tag satisfies "readable copy of attribution notices contained within such NOTICE file...within the Source form or documentation"

https://github.com/ben-manes/concurrentlinkedhashmap/blob/master/LICENSE#L111

Mostly not a fan of having license type files within the `java` source - converting to a docstring on a `package-info.java` would also be better to me

Reply:
> converting to a docstring on a `package-info.java` would also be better to me if needing to include it

went with this 👍 

I also re-generated the license report, which pulls in NOTICE files if they're in the dependencies jar file, but in this case it's not.